recipe/meta.yaml

```diff
@@ -30,24 +32,25 @@ test:
     - tests
   requires:
     - pytest >=6.2,<7
```

Comment:
`pytest` should be `>=7.2` https://github.com/graphql-python/graphql-core/blob/689a70d052a58277d20cc6b6348b0eb31ad3a113/pyproject.toml#L52

Reply:
Updated this